kpatch/kpatch

```diff
@@ -378,6 +378,10 @@ disable_patch () {
 
 	local enabled="$SYSFS/$modname/enabled"
 	if ! [[ -e "$enabled" ]]; then
+            if [[ -d "/sys/module/$modname" ]] ; then
```

Comment:
Tabs, not spaces please.  Otherwise looks good...

Reply:
OK, let me push a new version with tabs. Sorry about that.

Reply:
No worries... you can force push to the same branch.

Reply:
Done.

Reply:
Oh shellcheck... it can figure out the strangest single/double quotation compatibility bug, but not whitespace :)